### DISCUSSION TITLE
Blazor3D, ThreeJS, EyeShot

### DISCUSSION THREAD

**Discussion by reubenalfred** [Megabit/Blazorise#4829] [General]
Hi guys,

Blazor3D is now I believe is free to use and would be great to add 3D capability to Blazorise with some level of abstraction or perhaps start with support for a particular technology.

ThreeJS is open source as you know but EyeShot is pure c# and idiomatic but not free but referred here for ideas.

Would 3D capability make business sense for Blazorise? yes it would add to its use cases for visualisation in charts, 3d cad files and even games. And these features would open opportunites for different types of applications such as educational software with graphics to illustrate concepts and processes. Simulation software for work place training and so on...

**Comment by David-Moreira** [comment]
@reubenalfred opened a feature request. #4830 
Thanks for the suggestion.

**Comment by reubenalfred** [comment]
I used to work for a window fabrication place with an annual turnover of about AUD10 million. The problem they had on the factory floor was assembling skills. Fabricators had fundamental trade skills but not the ability to read tech drawings nor were the senior technicians willing to share their expertise with others because of self-preservation.

Imagine a Blazorise-based solution for this scenario: The order would describe all the details of the product/s that need to be fabricated. Required toolset for entire fabrication, BOM, stage-by-stage and step-by-step instructions, and proper orientation of the parts everything can be supported just by enabling 3D capability to Blazorise furthermore with a web app companies don't have to worry about installing on per device and maintenance. This method of assistance would make an individual self-reliant in most cases.

The sky is the limit! :)

**Reply by stsrki** [reply]
Considering we have some clients in the fabrication industry, I guess having some part of 3D capabilities makes sense.

**Comment by reubenalfred** [comment]
https://babylonblazorapp202208.azurewebsites.net/
